CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ else()
     cmake_minimum_required(VERSION 3.13.4)
 endif()
 
+set (CMAKE_CXX_STANDARD 11)
```

Comment:
Generally it's advised to set this on the cmake target rather than globally. It should be set for libcsound at least. Maybe the opcodes just need it set in make_plugin or something

Reply:
yes, I suspected that, but it did not seem to be defined because the CI failed for iOS. I think it's OK to use C++11 as our baseline though.